LibertyDSNP/example-client
# WIP Feat/remove unnecessary features #179011705

Purpose
---------------
Remove features that will not be used for the demo.
[Finishes #179011705](https://www.pivotaltracker.com/story/show/179011705)

Change summary
---------------
* Remove NewPost, NewPostImageUpload components and styles
* Hard code handle, name, and image URL (which may change)
* Disable inputs for Profile component
* Remove style guide

Steps to Verify
----------------
You should be able to build and run the app and click around, viewing the Feed without errors.

Additional details / screenshot
----------------
